CRM/Financial/BAO/ExportFormat.php

```diff
@@ -105,14 +103,6 @@ public function getFileExtension() {
   }
 
   /**
-   * Override this if appropriate.
-   * @return null
-   */
-  public function getTemplateFileName() {
-    return NULL;
```

Comment:
I'm assuming the parent does the same thing & this isn't intentionally blocking it?